### PR TITLE
copying abingdon docs to sheep

### PR DESCRIPTION
**Pull request by lomaxo**
Merging Abingdon docs with Hill's Road. Mostly need to keep Abingdon version but not always...

### CODE REVIEW

File: docs/expanding-functionality.md
Comment (lomaxo) [inline docs/expanding-functionality.md:1]: It looks like we may want to keep the systemetric version here which has added some detail about I2C.

File: docs/the-brainbox.md
Comment (lomaxo) [inline docs/the-brainbox.md:1]: Do we need to keep the systemetric PinOut notes?

File: docs/vision.md
Comment (lomaxo) [inline docs/vision.md:1]: Quite a lot of differences here but I assume we just keep Abingdon's version.

**Review by lomaxo — COMMENTED** [review]
@scorbett123 Would you be able to look through the comments I've made. It looks like the systemetric repo doesn't have the 2025 rules in it yet and we mainly need to keep Abingdon's changes everywhere but there are a few files that have differences we may need to keep.

File: docs/expanding-functionality.md
Comment (scorbett123) [reply docs/expanding-functionality.md:1]: I2C isn't available to the user on the pilow brains

File: docs/the-brainbox.md
Comment (scorbett123) [reply docs/the-brainbox.md:1]: The notes were removed as the images were outdated, if someone wants to recreate them then that would be great, however unless that is the case then I think it's best that they are removed.

File: docs/vision.md
Comment (scorbett123) [reply docs/vision.md:1]: I agree

**Review by scorbett123 — COMMENTED** [review]
I've left these comments here as they also apply to my next pr

**Pull request closed without merging**